documentation/assemblies/assembly-kafka-connect-authorization.adoc

```diff
@@ -0,0 +1,34 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-connect.adoc
+// assembly-deployment-configuration-kafka-connect-s2i.adoc
+
+[id='assembly-kafka-connect-authorization-{context}']
+
+= Kafka Connect user authorization
+
+If simple authorization is enabled for Kafka Connect, the `KafkaUser` resource for the Kafka Connect user must be configured to provide access rights to the consumer group and
```

Comment:
Some comments:
* This applies to any authorization, even the `keycloak` one. Only the examle in the procedure is for `simple` only
* We should clarify that both read and write access is needed.

Reply:
> If **authorization** is enabled for Kafka Connect, the `KafkaUser` resource for the Kafka Connect user must be configured to provide **read/write** access rights to the consumer group and internal topics, which are automatically configured or specified as configuration parameters in the `spec` for the `KafkaConnect` or `KafkaConnectS2I` configuration.